pkg/ingester/ingester_v2.go

```diff
@@ -835,6 +838,30 @@ func (i *Ingester) v2Push(ctx context.Context, req *cortexpb.WriteRequest) (*cor
 				return copiedLabels
 			})
 		}
+
+		// app.AppendExemplar currently doesn't create the series, it must
+		// already exist.  If it does not then drop.  TODO(mdisibio) - better way to handle?
+		if ref == 0 {
```

Comment:
Yes that is true.  We have two choices, we could count them as failed or discarded due to validation.  In this case I think failed is more appropriate because there isn't anything wrong with the exemplar itself, it is more that it couldn't be ingested due to a limitation in the current tsdb implementation.  If/when tsdb AppendExemplar is updated to create the series then the same data would be ingested successfully.